backend/README.md

```diff
@@ -352,6 +352,12 @@ You can run it this way with Docker:
     docker pull elasticsearch:7.0.0
     docker run -d -p 9200:9200 -p 9300:9300 -v esdata:/usr/share/elasticsearch -v esconfig:/usr/share/elasticsearch/config -e "discovery.type=single-node" -e "cluster.routing.allocation.disk.threshold_enabled=false" elasticsearch:7.0.0
 
+You need also a minio server on port 9000 for s» related test
+You can run it this way with Docker:
+
+    docker pull minio/minio
+    docker run -p 9000:9000 -e MINIO_ACCESS_KEY=minioadmin -e MINIO_SECRET_KEY=minioadmin minio/minio server /data
```

Comment:
I think we need to add some documentation about each parameter (to make sure not broken something on dev machine).

Reply:
@PhilippeAccorsi  i Added some explanation about the command, check if this is enough for you (i don't think we should go too deep into s3 stuff here, we just show docker to run for tests).